ui/src/Components/ProfilePage.jsx

```diff
@@ -94,13 +94,34 @@ export default class ProfilePage extends React.Component {
         return (
           <Form>
             <div className="right-align" style={{marginRight: '100px'}}>
-            <Link to= {`/profiles/edit/${this.state.profile.result.profile_id}`}className="btn btn-secondary">Edit this profile</Link>
+            <Link to= {`/profiles/edit/${this.state.profile.result.profile_id}`}className="btn btn-secondary">Edit Profile</Link>
             </div>
           </Form>
 
         )
       }
   }
+
+
+  if (this.state.profile && this.state.profile.resultRefs) {
+    const refs = this.state.profile.resultRefs;
+    var refLen = Object.keys(refs).length;
```

Comment:
Little nit: the concept of `vars` I think are now deprecate. I think you're supposed to use `let` in their place now.

Reply:
✔